content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,211 @@
+    ```
+
+1. Create a Signed Statement for the SPDX SBOM
+
+    ```shell
+    python scitt/create_signed_statement.py \
+      --signing-key-file $SIGNING_KEY \
+      --issuer $ISSUER \
+      --feed $FEED \
+      --content-type application/spdx+json \
+      --payload scitt/artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      --output-file signed-statement-sbom.cbor
+
+1. Register the SBOM
+
+    ```shell
+
+    ENTRY_ID=$(statement-register.py \
+      issuer $ISSUER \
+      signing-key $SIGNING_KEY \
+      feed $FEED \
+      payload artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      content-type application/spdx+json)
+    ```
+
+1. Retrieve a SCITT Receipt
+
+    ```shell
+    statement-get-receipt.py $ENTRY_ID
+    ```
+
+## Register an Attestation
+
+Create an Attestation that provides the artifacts compliance to the \<foo> specification
+
+1. Create the Attestation
+
+    ```shell
+    cat > attestation.json <<EOF
+    {
+      "compliance": [
+        {
+          "name": "standard-alpha",
+          "value": "true"
+        },
+        {
+          "name": "standard-beta",
+          "value": "false"
+        }
+      ]
+    }
+    EOF
+    ```
+
+1. Create a Signed Statement for the attestation
+
+    ```shell
+    python scitt/create_signed_statement.py \
+      --signing-key-file $SIGNING_KEY \
+      --issuer $ISSUER \
+      --feed $FEED \
+      --content-type application/json \
+      --payload attestation.json \
+      --output-file signed-statement-attestation.cbor
+
+1. Register the SBOM
+
+    ```shell
+    ENTRY_ID=$(statement-register.py \
+      issuer $ISSUER \
+      signing-key $SIGNING_KEY \
+      feed $FEED \
+      payload artifacts/_manifest/spdx_2.2/manifest.spdx.json \
+      content-type application/spdx+json)
+    ```
+
+## Retrieve Statements for the Artifact
+
+The power of SCITT is the ability to retrieve the history of statements made for a given artifact.
+By querying the series of statements, consumers can verify who did what and when for a given artifact.
+
+1. Query DataTrails for the collection of statements
+
+    ```shell
+    curl https://app.datatrails.ai/archivist/v2/publicassets/-/events?event_attributes.feed_id={{feed}}
+    ```
+
+To filter on specific content types, such as what SBOMs have been registered, or which issuers have made statements, see \<TODO: here>
+
+## Summary
+
+The quickstart created a collection of statements for a given artifact.
+Over time, as new information is available, authors can publish new statements which verifiers and consumers can benefit from.
+There are no limits to the types of additional statements that may be registered, which may include new vulnerability information, notifications of new versions, end of life (EOL) notifications, or more.
+By using the content-type parameter, verifiers can filter to specific types, and/or filter statements by the issuer.
+
```

Comment:
I think it would be nice to add a small statement and link to the quickstart to verify a scitt receipt. (We still have to write that!)